Core/FeatureFlag.swift

```diff
@@ -68,7 +69,9 @@ extension FeatureFlag: FeatureFlagSourceProviding {
             return .remoteReleasable(.subfeature(AutoconsentSubfeature.onByDefault))
         case .history:
             return .remoteReleasable(.feature(.history))
-        
+        case .historyRollout:
```

Comment:
We needed a separate feature so that we can have a sub feature for rolling out since the previous feature was already used.  Once at 100% we'll switch to just the history feature.